bitnami/redis/values.yaml

```diff
@@ -23,6 +23,12 @@ global:
 ## @section Common parameters
 ##
 
+## @param externalDNS Boolean to indicate the desire to utilize `external-dns`
+##
+externalDNS:
+  enabled: false
+  # suffix: ""
+
```

Comment:
I'm not sure this is a good way to name the parameters, as users could assume that this also deploys a installation of the `bitnami/external-dns` chart.

Reply:
I would agree.  I've renamed it to `useExternalDNS`.  Though I'm not thrilled about that name either.  Open to further suggestions.